modules/hcp-bm-hc.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-bm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-bm-hc_{context}"]
+= Creating a hosted cluster on bare metal
+
+When you create a hosted cluster with the Agent platform, HyperShift installs the Agent Cluster API provider in the hosted control plane namespace. You can create a hosted cluster on bare metal or import one.
+
+As you create a hosted cluster, keep the following guidelines in mind:
+
+- Each hosted cluster must have a cluster-wide unique name. A hosted cluster name cannot be the same as any existing managed cluster in order for {mce-short} to manage it.
+
+- Do not use `clusters` as a hosted cluster name.
+
+- A hosted cluster cannot be created in the namespace of a {mce-short} managed cluster.
+
+.Procedure
+
+. Create the hosted control plane namespace by entering the following command:
++
+[source,terminal]
+----
+$ oc create ns <hosted_cluster_namespace>-<hosted_cluster_name>
+----
++
+Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted_cluster_name>` with your hosted cluster name.
+
+. Verify that you have a default storage class configured for your cluster. Otherwise, you might see pending PVCs. Run the following command:
++
+[source,terminal]
+----
+$ hcp create cluster agent \
+    --name=<hosted_cluster_name> \ <1>
```

Comment:
Callouts in multi-line commands need to be commented out per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks).

Example:

```
[source,terminal]
----
$ oc autoscale <object_type>/<name> \// <1>
```